src/views/_base.njk

```diff
@@ -48,6 +48,8 @@
 
 {% block head %}
   <link rel="stylesheet" href="{{pathPrefix}}/dist/main.css">
+  <link rel="stylesheet" href="{{pathPrefix}}/accessible-autocomplete/accessible-autocomplete.min.css">
+  <script src="{{pathPrefix}}/accessible-autocomplete/accessible-autocomplete.min.js"></script>
```

Comment:
Did these need to be moved here?

By including them in the `_base` file, they'll be loaded on every page load (including the pre-login ones).